packages/bisect_ppx/bisect_ppx.0.2.2/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "bisect_ppx"
+version: "0.2.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bisect_ppx"
+dev-repo: "https://github.com/rleonid/bisect_ppx.git"
+bug-reports: "https://github.com/rleonid/bisect_ppx/issues"
+build: [
+  ["sh" "configure"]
+  [make "all"]
+  [make "install"]
+]
+depends: [
+  "ocamlfind" "ppx_tools" {build}
```

Comment:
you might want to have:

```
"ocamlfind" {build}
"ppx_tools" {build}
```

the `build` filter is currently only attached to `ppx_tools`